code/modules/antagonists/malf_ai/malf_ai_modules.dm

```diff
@@ -919,7 +923,7 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module/malf))
 /datum/ai_module/malf/upgrade/voice_changer
 	name = "Voice Changer"
 	description = "Allows you to change the AI's voice. Upgrade is active immediately upon purchase."
-	cost = 40
+	cost = 20
```

Comment:
Is this a balance change or just being changed to be the same under the new resource economy?